code/osapi/DebugWindow.cpp

```diff
@@ -0,0 +1,175 @@
+//
+//
+
+#include "DebugWindow.h"
+
+#include "globalincs/alphacolors.h"
+
+namespace {
+uint32_t get_debug_display() {
+	// If there are two or more monitors then we would like to display the window on the second monitor but it shouldn't
+	// be shown on the same monitor that the game is running on
+	// If there is only then then we'll just display it on that one
+	auto numDisplays = SDL_GetNumVideoDisplays();
+	
+	if (numDisplays < 1) {
+		// Some kind of error
```

Comment:
Not sure what the proper error handler should be. Int3() is the first to come to mind, but I don't think that's the best.

Could put "TODO" in the comment to serve as a marker if we decide to defer putting in the error handler.